docs/configuration.rst

```diff
@@ -237,7 +237,19 @@ See :ref:`the corresponding tutorial <theming>`.
 Installing extra xblocks and requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Would you like to include custom xblocks, or extra requirements to your Open edX platform? Additional requirements can be added to the ``env/build/openedx/requirements/private.txt`` file. For instance, to include the `polling xblock from Opencraft <https://github.com/open-craft/xblock-poll/>`_::
+Would you like to include custom xblocks, or extra requirements to your Open edX platform? Additional requirements can be added to the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` parameter in the :ref:`config file <configuration>` or to the ``env/build/openedx/requirements/private.txt`` file. For instance, to include the `polling xblock from Opencraft <https://github.com/open-craft/xblock-poll/>`_:
+
+- add the following to the ``config.yml``::
+
+    OPENEDX_EXTRA_PIP_REQUIREMENTS:
+    - "git+https://github.com/open-craft/xblock-poll.git"
+
+.. warning::
+   Specifying extra requirements through ``config.yml`` overwrites the default
+   extra requirements. You might need to add them to the list, if your
+   configuration depends on them.
+
+- or add the dependency to ``private.txt``::
```

Comment:
We should explain what is the point of adding a dependency to private.txt versus adding it to OPENEDX_EXTRA_PIP_REQUIREMENTS. private.txt is for private requirements that need to be cloned locally because we don't want the git credentials to be added to the Docker image.

Reply:
Done :+1: